Packs/SophosCentral/Integrations/SophosCentral/SophosCentral.py

```diff
@@ -2006,7 +2009,9 @@ def sophos_central_endpoint_list_command(client: Client, args: dict) -> CommandR
         args.get("ids"),
         args.get("view"),
         min(int(args.get("limit", "")), 100),
+        argToList(args.get("ip_addresses")),
     )
+    full_match_ip = args.get("full_match_ip") == "true" if args.get("full_match_ip") else None,
```

Comment:
The setting of `full_match_ip` is a little confusing as it calls `args.get("full_match_ip")` twice.
I suggest using CommonServerPython's `argToBoolean` function for a much more complete check, like so:
```suggestion
    full_match_ip = True if argToBoolean(args.get("full_match_ip")) else None,
```